platform/server/elasticsearch/Cluster.ts

```diff
@@ -13,7 +13,7 @@ export class Cluster {
     private readonly config: ElasticsearchConfig,
     logger: LoggerFactory
   ) {
-    this.log = logger.get('elasticsearch', 'client', config.clusterType);
+    this.log = logger.get(`elasticsearch::client::${config.clusterType}`);
```

Comment:
I agree that having comma separated arguments for namespace part looks better than making consumers to learn our namespace separator.... Maybe we can provide public api that accepts such format, but internally joins everything into a string, or something like this not sure right now. It will become more obvious what we need when we use it for some time.